conf/airframes/examples/bebop.xml

```diff
@@ -10,6 +10,17 @@
       <module name="udp"/>
     </target>
 
+    <define name="MT9F002_OUTPUT_HEIGHT" value="640" />
```

Comment:
Ok. So the output width/height is just the cropped part?? Or downsampled?

In any case most people won't be able to set all these defines appropriately.
So I guess it would be nice if we had a couple of useful pre-defined combinations that you can then select (define).